package.json

```
@@ -24,7 +24,7 @@
   },
   "homepage": "https://github.com/axic/swarmhash#readme",
   "dependencies": {
-    "keccakjs": "^0.2.3",
+    "keccak": "^1.4.0",
```

It's already at 3.0.2:
```suggestion
    "keccak": "^3.0.0",
```